{{cookiecutter.project_dash_slug}}/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Entrypoint script
+
+main() {
+  [ $# -lt 1 ] && pipenv_shell
+  case "$1" in
+    -*) pipenv_shell "$@";;
+    *) exec "$@";;
+  esac
+}
+
+pipenv_shell() {
+  pipenv shell
```

Comment:
Question: what's the goal of the Docker image? If it's not just for development, we should make the default command something like `pipenv run "$@"`.

Reply:
The goal is to run the python module developed and exit. In the case of de-identification for example something like `python deidentify --customer miami`

Reply:
Also this script I was kind of flying blind in the effort to make it generic. your suggestion is good.